frontend/src/Order.tsx

```diff
@@ -177,76 +177,47 @@ const Order = () => {
                             <></>
                         )}
                     </Grid>
-                    <Grid item container xs={12} spacing={3} data-testid={'personal_device'}>
-                        <Grid item xs={10} sm={3}>
-                            <FieldHeader headerText={'Device type'} />
-                            <Radio
-                                label="Personal device"
-                                value="personal"
-                                checked={deviceType === 'personal'}
-                                onChange={() => {
-                                    setSingleField('deviceType', 'personal')
-                                }}
-                            />
-                        </Grid>
-                        <Grid item xs={10} sm={3} data-testid={'multi_user_device'}>
-                            <FieldHeader headerText={'\u00a0'} /> {/*Unicode for non-breaking space in order to align headers*/}
-                            <Radio
-                                label="Multi-user device"
-                                value="multi"
-                                checked={deviceType === 'multi'}
-                                onChange={() => {
-                                    setSingleField('deviceType', 'multi')
-                                }}
-                            />
-                        </Grid>
-                    </Grid>
-                    {deviceType === 'personal' ? (
-                        userType === 'Equinor personnel' ? (
-                            //Personal equinor employee device
-                            <>
-                                <Grid item container xs={12} spacing={3} alignItems="center">
-                                    <Grid item xs={10} sm={5} data-testid={'user_type_dropdown'}>
-                                        <FieldHeader headerText={'User type'} />
-                                        <Select options={userTypeOptions} onSelect={item => setSingleField('userType', item.title)} />
-                                    </Grid>
-                                    <HelpIcon helpText={'info text'} />
+                    {userType === 'Equinor personnel' ? (
+                        //Personal equinor employee device
```

Comment:
Personal can be removed from these comments

Reply:
Fixed